ASCIIEngine/Components/Component.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace ASCIIEngine.Core.Components
+{
+    public class Component
+    {
+        public virtual void Update(){}
```

Comment:
Where's Start() method?

Reply:
It's not necessary at this implementation